### ISSUE TITLE
Reference/authority for acepted taxa name

### ISSUE THREAD

**Issue by gpopac** [Biologer/Biologer-Taxonomy#35]
It would be good to add the reference or ID in the data source used for the final name of the taxon. I.e. reference of taxonomic paper describing new subspecies or ID from plant med database.

**Comment by ekoblogist** [comment]
Yes, in order to enable users to use Biologer data in various scenarios it is important to know exactly to which taxonomical reference taxon names are resolving. In the case of vascular plants, I think taxonomy is connected to euro+med database, however, this should be double-checked, because there are two versions of this taxonomic reference which can not be mapped 1 on 1 since they are not using the same IDs (there is not even a reference from newer to older one). For example, ID for Fagus sylvatica L. in the first version of euro+med (http://ww2.bgbm.org/EuroPlusMed/query.asp) is 23372, while in the latest version (https://europlusmed.org/) it is 68c1ac04-391b-49df-990a-3dd6a75d05b6.
